clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/api/v1/services/AgentApi.groovy

```diff
@@ -49,6 +49,6 @@ interface AgentApi {
   @PUT("/v1/agent/service/deregister/{serviceId}")
   Response deregisterService(@Path("serviceId") String serviceId)
 
-  @PUT("/v1/agent/service/maintenance/{serviceId}")
-  Response maintenance(@Path("serviceId") String serviceId, @Query("enable") boolean enable, @Query("reason") String reason)
+  @PUT("/v1/agent/maintenance")
```

Comment:
The reason I changed this is because there are two `maintenance` endpoints, and the one dealing with specific services isn't useful to us.